Makefile

```diff
@@ -22,6 +22,7 @@ ETCDIR ?= /etc
 TMPFILESDIR ?= ${PREFIX}/lib/tmpfiles.d
 SYSTEMDDIR ?= ${PREFIX}/lib/systemd/system
 USERSYSTEMDDIR ?= ${PREFIX}/lib/systemd/user
+REMOTETAGS := !ABISupport remoteclient exclude_graphdriver_btrfs btrfs_noversion  exclude_graphdriver_devicemapper containers_image_openpgp
```

Comment:
Should we use the hack/ scripts to determine whether btrfs, seccomp, etc are included?

Reply:
No, We don't want this code compiled into remoteclient. In a perfect world we would not hit this.  The btrfs and devicemapper flags are for containers/image and containers/storage and prevent btrfs and devicemapper from being compiled into the client.